controllers/storagecluster/topology.go

```diff
@@ -23,6 +23,13 @@ import (
 
 type ocsTopologyMap struct{}
 
+var validTopologyLabelKeys = []string{
+	corev1.LabelTopologyRegion,
+	corev1.LabelTopologyZone,
+	corev1.LabelHostname,
+	defaults.RackTopologyKey,
```

Comment:
If other [Rook topology labels](https://rook.io/docs/rook/latest-release/CRDs/Cluster/ceph-cluster-crd/#osd-topology) (e.g. room, row, pdu) are found on the nodes, they will just be ignored by ODF, right?

Reply:
Yes, ODF only supports 3 failure domains zone, host, rack. So any other labels found on the Nodes would be ignored.
https://github.com/red-hat-storage/ocs-operator/blob/164f7c4c3b997e5e2c66cb3fdebcb91a8d55d6d2/api/v1/topologymap.go#L69-L74